### PR TITLE
Add ability to retreive grpc service instance from context

### PR DESCRIPTION
**Pull request by isengrims**

### CODE REVIEW

File: src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/impl/HandlerContext.kt [hunk 1]
```diff
@@ -32,6 +33,7 @@ class HandlerContext(
     private val channelFactory: ChannelFactory,
     private val getDictionary: (DictionaryType) -> InputStream,
     private val sendEvent: (Event) -> Unit,
+    private val grpcRouter: GrpcRouter
```
Comment (ivandruzhinin) [inline src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/impl/HandlerContext.kt:36]: Let's pass a method reference instead of the whole router

**Review by ivandruzhinin — COMMENTED** [review]
Please also bump version and update changelog in readme

File: src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/impl/HandlerContext.kt (hunk 1 above)
Comment (isengrims) [reply src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/impl/HandlerContext.kt:36]: How? We don't have common interface for service classes

Comment (ivandruzhinin) [reply src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/impl/HandlerContext.kt:36]: > How?

We can do smth like this, but we'll have to do an unchecked cast later

```suggestion
    private val getService: (Class<out Any>) -> Any
```

File: README.md [hunk 2]
```diff
@@ -138,6 +138,11 @@ spec:
 
 # Changelog
 
+## 2.1.0
+
+* Handler context interface changed: added option to retrieve grpc service instances.
+* JavaTimeModule added to object mapper
```
Comment (ivandruzhinin) [inline README.md:144]:
```suggestion
* allow to retrieve gRPC service from handler context
* support JSR-310 date and time types in settings
```

**Review by ivandruzhinin — APPROVED** [review]
**Please merge with squash**

**Pull request merged**